contracts/CommunityTokenHolder.sol

```diff
@@ -22,7 +22,7 @@ contract DevTokensHolder is Controlled {
     uint256 balance = aix.balanceOf(address(this));
     uint256 total = collectedTokens.add(balance);
     // This wallet will get a 29% of the total tokens.
-    // since scaling 4 of 29 to a percentage looks horrible (13.7931034483),
+    // since scaling 7 of 29 to a percentage looks horrible (24.1379310344828),
```

Comment:
👍 great commitment on updating also the comments

Reply:
👍 :)